cmd/deploy.go

```diff
@@ -75,12 +76,9 @@ func (c *DeployCommand) Run(args []string) int {
 
 	extraVars := fmt.Sprintf("\"%s\"", strings.Join(vars, " "))
 
-	playbookArgs := []string{"deploy.yml", "-e", extraVars}
-	deploy := execCommand("ansible-playbook", playbookArgs...)
-	logCmd(deploy, c.UI, true)
-	err := deploy.Run()
+	c.playbook.SetRoot(c.Trellis.Path)
 
-	if err != nil {
+	if err := c.playbook.Run("deploy.yml", []string{"-e", extraVars}); err != nil {
```

Comment:
This breaks `extraVars` somehow 🤔 Ansible doesn't recognize the actual vars within `-e`

Reply:
Actually it might have been broken before somehow? Trying to figure it out

Reply:
Fixed in master